apidoc/Titanium/UI/UI.yml

```diff
@@ -1203,312 +1203,250 @@ properties:
     platforms: [android, iphone, ipad]
 
   - name: BLEND_MODE_CLEAR
-    deprecated:
-        since: "1.8.0"
-        notes: Use <Titanium.UI.iOS.BLEND_MODE_CLEAR> instead.
-        removed: "6.0.0"
     summary: |
         Use with [MaskedImage.mode](Titanium.UI.MaskedImage.mode) to specify a blend mode.
     type: Number
     permission: read-only
-    platforms: [iphone, ipad]
+    since: "7.2.0"
+    platforms: [android, iphone, ipad]
 
   - name: BLEND_MODE_COLOR
-    deprecated:
-        since: "1.8.0"
-        notes: Use <Titanium.UI.iOS.BLEND_MODE_COLOR> instead.
-        removed: "6.0.0"
     summary: |
         Use with [MaskedImage.mode](Titanium.UI.MaskedImage.mode) to specify a blend mode.
+    description: |
+        Android does not support this blend mode.
     type: Number
     permission: read-only
-    platforms: [iphone, ipad]
+    since: "7.2.0"
+    platforms: [android, iphone, ipad]
```

Comment:
If it is not supported on Android, I would just leave out the `android` key. If we would be very exact here, we would leave the iOS only constants on the iOS namespace and put the parity ones here.

Reply:
I still define the constant on Android, but log a warning instead. I think it's better than letting the app crash.

It is possible to add the other modes on Android, but we would have to manipulate the bitmap pixels ourselves the slow way. (yuck)

I'll remove the "android" platform keys from the unsupported constants.